src/js/components/DataTable/Header.js

```diff
@@ -300,7 +300,13 @@ const Header = forwardRef(
               )}
             </StyledDataTableCell>
           )}
-          {rowDetails && <TableCell size="xxsmall" plain pad="none" />}
+          {rowDetails && (
+            <TableCell
+              size={theme.dataTable.rowDetails?.size}
```

Comment:
General naming question -- it seems like this and the footer groupBy one are all related to how to size a cell when there is an "expand" cell. Is that correct?

If so, seems like all those would move in unison -- should we simplify to `theme.dataTable.expand.size` ? or something

Reply:
Yeah I like `theme.dataTable.expand.size`, I'll update to use that